composer/trainer/_patch_pytorch.py

```diff
@@ -109,6 +109,12 @@ def patch_pytorch():
         # No monkeypatches!
         pass
 
+    elif version.parse(torch.__version__) < version.parse('2.5.1'):
+        # Monkey patch for torch < 2.5.1 ie torch == 2.5.0
+
+        # No monkeypatches!
```

Comment:
we should prob make htis comment more clear that says besides the ones for unshard

Reply:
Done